articles/overview/how-to-learn-quantum-computing.md

```diff
@@ -0,0 +1,46 @@
+---
+title: How to learn quantum computing
+description: 
+author: natke
+ms.author:  nakersha
+ms.date: 10/16/2019
+ms.topic: article
+uid: microsoft.quantum.overview.learn
+---
+
+# How to learn quantum computing
+
+Learn how to write your first quantum program.
+
+## Getting the basics: what do you need to know?
+
+Fortunately, you don’t need to know quantum physics to start writing quantum applications. However, there are some concepts that are especially important to understand quantum computing. But don’t worry if you don’t know something! We will give you the resources to learn the necessary tools in a light and interactive way.
+
+* Linear algebra (vector and matrices): quantum states are vectors in a vector space over the complex numbers. Quantum gates are just linear operations acting on these states. Here you can find a tutorial to get everything you need to know about linear algebra. (Link to a Binder for the tutorial on linear algebra).
+* Complex numbers: quantum mechanics makes use of complex numbers to express quantum states. They are crucial to understand the basics of quantum computation. Here you can find a tutorial to get everything you need to know about complex numbers.
+* Basic quantum mechanics: ok, I know we just said that you don’t need to know quantum physics to start coding (and it’s true!). But some basic concepts of quantum mechanics and its mathematical notation will be helpful to understand quantum programming. Here you can find a tutorial to get everything you need to know about quantum mechanics. (Link to a Binder for the tutorial on basic quantum mechanics).
```

Comment:
Complex arithmetic tutorial is up at https://mybinder.org/v2/gh/Microsoft/QuantumKatas/master?filepath=tutorials/ComplexArithmetic/ComplexArithmetic.ipynb. The others are still in review, should be up some time this week - I'll let you know.

By the way, I would swap complex numbers and linear algebra in the text, since complex numbers typically come first in the learning path.

Reply:
There's a number of different learning paths that readers could follow; I don't think complex numbers come first in all or even most of them?